yarn.lock

```diff
@@ -7212,6 +7212,13 @@
   dependencies:
     "@backstage/backend-plugin-api" "^0.7.0"
 
+"@janus-idp/backstage-plugin-topology-common@1.4.0":
+  version "1.4.0"
+  resolved "https://registry.yarnpkg.com/@janus-idp/backstage-plugin-topology-common/-/backstage-plugin-topology-common-1.4.0.tgz#947f33e30a22a03e17d5c850db769d09fd45536e"
+  integrity sha512-BSGz/tvufixTOpUN11r0oi1cQGZtS0JGMoNGkOLJy+O3BbdYECsZewq5CXCFYr+l7VJEwJBdsVnDztb+Sz/OJw==
+  dependencies:
+    "@backstage/plugin-permission-common" "^0.8.0"
+
```

Comment:
is this a change related to this PR? I **think** we should fix it in a different PR if it derives from main branch

Reply:
i had failures in CI due to yrn.lock. trying to fix it